src/docs/product/data-management-settings/dynamic-sampling/getting-started.mdx

```diff
@@ -22,60 +20,20 @@ The **Dynamic Sampling** feature is currently only available for the following S
-
-![A sampling rule with the Active toggle highlighted](sampling-rule-toggle.png)
-
-## 3. Set a Sampling Rate Based on a Condition
-
-Now, click “Add Rule” to set more server-side sampling rules based on specific conditions. These rules will help you target the transactions you want to keep and allow you to have more of those transactions sent by the SDK.
-
-When a rule uses multiple conditions, we use the logical expression `AND` to determine when that rule should be evaluated. Multiple values for a condition use a logical or condition for matching. Learn more about [how to create effective server-side sampling rules](/product/data-management-settings/dynamic-sampling/sampling-configurations/).
+## 2. Update Data Retention Priorities
 
-You can use the following conditions to sample transactions by:
+While we do our best to provide you with the most useful transaction data, our **Dynamic Sampling** feature allows you to turn the priorities that we apply to your data retention on or off:
 
-- Environment
-- Release - Allows glob pattern matching and includes the option to sample by the "Latest" release
+![The Dynamic Sampling page with priority toggles enabled.](dynamic-sampling.png)
 
-![Adding a sampling rule based on a release condition](add-sampling-rule.png)
+Because every project is different, you can set these toggles differently for different projects. For example, you may have a project where the retaining transactions from the latest release is not a priority for you.
 
-If we've accepted a transaction which contains a key-value pair matching the above conditions in the past, Sentry will suggest values for each condition. Of course, you're free to set a custom value for each condition. Then, specify a sampling rate for that specific rule.
+By default, these toggles are all enabled so that Sentry's dynamic sampling priorities are applied to your data.
 
-## 4. Increase Your SDK Transaction Sample Rate
+## 3. Increase Your SDK Transaction Sample Rate {#4-increase-your-sdk-transaction-sample-rate}
 
-Generally speaking, the more active sampling rules you have, the fewer transactions we'll accept. If the volume of transactions our servers are receiving is insufficient given the rules you've specified, you might need to increase the sampling rate configured for your Sentry SDK. To see what we would suggest for your SDK sample rate, go to **[Project] > Settings > Dynamic Sampling** and open the last rule in the list, which is where your uniform sample rate is set. Our suggestion is displayed there.
+Our dynamic sampling works best when you send us as much data as you can (ideally 100% but we'll take what you can give us). As you purchase more transaction volume however, your pricing does down accordingly and you see significant savings. If we don't think we're able to provide you with good quality data because your client-side sampling rate is too low, we'll let you know.
```

Comment:
```suggestion
Our Dynamic Sampling works best when you send us as much data as you can (ideally 100% but we'll take what you can give us). As you purchase more transaction volume, however, your pricing does down accordingly and you see significant savings. If we don't think we're able to provide you with good-quality data because your client-side sampling rate is too low, we'll let you know.
```